README.md

```diff
@@ -9,9 +9,8 @@ These widgets are built using Dojo's widget authoring system [(@dojo/framework/c
 
 - [Usage](#usage)
 - [Features](#features)
+- [Examples](#examples)
```

Comment:
What if we have two top-level headings targets towards consumers and the other targeted towards contributors. Something like `Using Widgets` and `Contributing`? As a consumer, I usually stop reading once I've reached the part of the README that pertains to contributors.